### DISCUSSION TITLE
openURL

### DISCUSSION THREAD

**Discussion by justme-1968** [elgatosf/elgato#103] [Help]
i'm trying to open a deep link into an application from my plugin on a mac. but this does not seem to work.

opening a url like https://www.elgato.com works and opens the url in safari.

opening a url like <appName>://x-callback-url/homekit/1CA5037D... does not work

are schemes other than http and https supported with openURL? using the same urls with the system/website action do work.

but the get in background checkbox has to be disabled. if it is enabled it also does not work.
what does the get in background toggle exactly do?

**Comment by ChekTek** [comment]
Hi @justme-1968, 

Unfortunately, schemas other than `http` or `https` are not supported at this time. You are not the first developer to have this request! I will discuss this feature request with the Stream Deck team. ✅

**Reply by justme-1968** [reply]
ok. thanks.

in the meantime i have switched to a platform native call to open the url. this works fine for now in combination with didReceiveDeepLink for the response handling.
